articles/anomaly-detection/passwords-checking.md

```diff
@@ -0,0 +1,229 @@
+Bulk Passwords Checking
+=======================
+
+In Auth0 we focus on bring you the tools you need to keep your systems and your users safe.
+Bulk passwords checking is a tool that allows you to check your user's passwords against pre-defined list of
+passwords to test their strength.
+
+This tool together with our [Breached Password Security](/anomaly-detection/breached-passwords) helps you audit your
```

Comment:
This tool, together with our [Breached-Password Security](/anomaly-detection/breached-passwords) functionality, helps you audit your users' security and take appropriate actions to protect them and their accounts.